### PR TITLE
update types for launchpad@v0.6.0

### PR DESCRIPTION
**Pull request by FredKSchott**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/bitovi/launchpad/compare/0.5.4...v0.6.0
- [X] Increase the version number in the header if appropriate: N/A
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`: N/A

### CODE REVIEW

**Comment by rictic** [comment]
As this is a major version with a breaking change it looks like we should follow this protocol: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

**Comment by FredKSchott** [comment]
@rictic Good catch! I hadn't seen that mentioned. Updating now...

**Comment by FredKSchott** [comment]
@rictic hmm, it looks like those instructions only apply to true major versions. v0.X is not a folder name that DT's `npm test` understands, and so [tests fail](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/228551759) as:

```
Error: Directory /Users/fschott/Code/DefinitelyTyped/types/launchpad/v0.5 has unused file index.d.ts
    at /Users/fschott/Code/DefinitelyTyped/node_modules/types-publisher/src/lib/definition-parser.ts:349:11
    at next (native)
    at fulfilled (/Users/fschott/Code/DefinitelyTyped/node_modules/types-publisher/bin/lib/definition-parser.js:4:58)
```

Reverting that change, @rictic PTAL
& /cc @mhegazy who just responded to a similar issue in #15022 to confirm.

**Comment by mhegazy** [comment]
> & /cc @mhegazy who just responded to a similar issue in #15022 to confirm.

yes, only major versions are supported (i.e. first part of the version).

**Comment by rictic** [comment]
Hm, but v0.6.0 is a major breaking change from v0.5.0 in the same way that v6.0.0 would be from v5.0.0 (and indeed, this PR will break users that are using the v0.5.x `remote` function).

File: types/launchpad/index.d.ts [hunk 1]
```diff
@@ -34,16 +34,6 @@ export function browserstack(
     authCreds: BrowserstackAuth,
     cb: (err: any, browserstack: Launcher) => void): void;
```
Comment (rictic) [inline types/launchpad/index.d.ts]: Please add version information to the header. i.e.

```typescript
// Type definitions for launchpad v0.6.0
```

**Comment by rictic** [comment]
Hm, but maybe adding the version to the header will handle that? If this is published as v0.6.0 of `@types/launchpad` then we'll be fine, as the current release is `v0.0.4`

**Review by rictic — CHANGES_REQUESTED** [review]

**Comment by FredKSchott** [comment]
@rictic oh nice, I had assumed that was sugar. Incorrect! https://github.com/DefinitelyTyped/definition-header/blob/master/src/serialise.ts

It also looks like the current types were properly published as `v0.0.4`, so I don't think that adding the header version in the same PR as the breaking changes will overwrite what's already been published.

Sounds good, thanks for double-checking @rictic

**Review by rictic — APPROVED** [review]
👍

**Comment by rictic** [comment]
👍 

in case the bot doesn't see reviews

**Pull request merged**